ultralytics/models/sam/modules/sam.py

```diff
@@ -58,8 +58,8 @@ def __init__(
         image_encoder: ImageEncoderViT,
         prompt_encoder: PromptEncoder,
         mask_decoder: MaskDecoder,
-        pixel_mean: list[float] = (123.675, 116.28, 103.53),
-        pixel_std: list[float] = (58.395, 57.12, 57.375),
+        pixel_mean: list[float] = [123.675, 116.28, 103.53],
+        pixel_std: list[float] = [58.395, 57.12, 57.375],
```

Comment:
@onuralpszr problem here is Python now wants immutable arguments, so I changed these form list to tuple a while back. A little annoying but means that we're not supposed to had default args as lists.

Reply:
I will revert okay 👍